### ISSUE TITLE
DNS no longer stops CMA update prompt

### ISSUE THREAD

**Issue by RoyalNexus** [hacks-guide/Guide_Vita#28]
So, for one reason or another, the DNS to block update prompts (212.47.229.76, featured on the installing h-encore page) for the vita doesn't seem to work anymore (At least in CMA's case). This is a problem because CMA will try and force the user to update to an unhackable firmware in order to transfer over the exploit.

That said, there's an easy way around this, by simply turning on airplane mode and rebooting the Vita, it will no longer check for updates. Allowing users to use CMA freely. However, if this problem isn't specific to CMA (i.e. CMA pings an update server not blocked by the DNS) and spans the entire system instead, it may cause users to accidentally update.

Suggestions:

- Check to see if the problem is simply unique to CMA with 212.47.229.76, and append the installing h-encore page with the airplane mode fix (alternatively, seek out the host of the DNS and ask them to block whatever CMA pings).
- Check to see whether the DNS on the blocking updates page (23.96.6.207) works (I'm guessing this points to 3.69 as the latest update, though).

**Comment by Plailect** [comment]
As the blocking update page says, the second DNS points to a 3.65 update server and is used primarily for updating from <3.65 to 3.65. As for the CMA DNS issue, I am still on 3.68 so I will test this (since I am on non-latest). I find it hard to believe that this could have somehow stopped working for everyone for no reason, and I suspect it is just a weird issue on your end.

**Comment by RoyalNexus** [comment]
Lots of users in the vita hacking discord are now reporting that the 3.65 update server is in-fact down.

https://i.imgur.com/6f6OaMw.png
https://i.imgur.com/uxQTOys.png
https://i.imgur.com/ac75sil.png

Seems like the best way to go about this is to just create a guide on how to update manually, like referenced in issue #30. The only remaining problem is how to stop the vita from auto-updating, is unchecking “Download Update File for System Software” is system settings not enough?

**Comment by Plailect** [comment]
Closing in favor of #33.
